.golangci.yaml

```diff
@@ -24,3 +24,8 @@ issues:
       linters:
         - dupl
         - funlen
+
+linters-settings:
```

Comment:
This setting turns off this type of error from the linter:
https://github.com/polyfloyd/go-errorlint#fmterrorf-wrapping-verb
I think we want this setting because we don't always want our APIs to include other dependencies' errors.